src/utils/createReactJestConfig.ts

```diff
@@ -0,0 +1,5 @@
+export default function createReactJestConfig(resolve: Function, projectRoot: string, ejected: boolean, loader?: NodeRequire): string {
```

Comment:
Do I understand correctly that this function is specifically targeted at people who use [Create React App](https://github.com/facebookincubator/create-react-app)? How about also supporting users of its [TypeScript counterpart](https://github.com/wmonk/create-react-app-typescript)?

Reply:
Yes it is targetted towards the users that use `create-react-app` to create their react apps. Adding support for the typescript counterpart sounds good. I'd have to investigate this though. It might be better to merge this and make an issue for typescript support as I am not sure I can add this in the last two weeks of my graduation.

Reply:
People can always eject their project to get a jest config via the `package.json`, I realise this isn't ideal because you'd lose all functionality that is provided by `react-scripts` but it's a way to mutation-test the project.